x-pack/legacy/plugins/uptime/public/components/functional/monitor_list/monitor_list_drawer/__tests__/integration_group.test.tsx

```diff
@@ -41,12 +41,12 @@ describe('IntegrationGroup', () => {
   it('will not display infra links when infra is unavailable', () => {
     const component = shallowWithIntl(
       <IntegrationGroup
-        basePath="foo"
-        dateRangeStart="now-12m"
-        dateRangeEnd="now-1m"
-        isApmAvailable={true}
-        isInfraAvailable={false}
-        isLogsAvailable={true}
+        // basePath="foo"
```

Comment:
Delete this code if it's not longer needed.

Reply:
done